crypto/s2n_ecdsa.c

```diff
@@ -32,8 +35,27 @@
 #include "crypto/s2n_openssl.h"
 #include "crypto/s2n_pkey.h"
 
+
 #define S2N_ECDSA_TYPE 0
 
+EC_KEY *s2n_unsafe_ecdsa_get_non_const(const struct s2n_ecdsa_key *ecdsa_key) {
+    if(ecdsa_key == NULL) {
+        return NULL;
+    }
+
+    /* pragma gcc diagnostic was added in gcc 4.6 */
+#if defined(__clang__) || S2N_GCC_VERSION_AT_LEAST(4,6,0)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wcast-qual"
+#endif
+    EC_KEY *out_ec_key = (EC_KEY *) ecdsa_key->ec_key;
+#if S2N_GCC_VERSION_AT_LEAST(4,6,0)
```

Comment:
You're missing clang

Reply:
https://github.com/aws/s2n-tls/pull/3474/commits/52c5df3534746ae4e4ae9b48c5e3e84f44508996